contracts/SimpleSwap.sol

```diff
@@ -11,7 +9,7 @@ contract SimpleSwap is SW3Utils {
   event Deposit(address depositor, uint amount);
   event ChequeCashed(address indexed beneficiary, uint indexed serial, uint amount);
   event ChequeSubmitted(address indexed beneficiary, uint indexed serial, uint amount);
-  event ChequeBounced(address indexed beneficiary, uint indexed serial, uint paid, uint bounced);
```

Comment:
I omitted the possibility for a cheque to be partially fulfilled; it bounces or not at all. Therefore, these variables can be omitted